rel/overlay/etc/default.ini

```diff
@@ -930,3 +930,97 @@ url = {{nouveau_url}}
 ;max_objects = 10000
 ;max_idle = 600000
 ;enable = true
+
+[couch_scanner]
+; How often to check for configuration changes and start/stop plugins
+;interval_sec = 5
+
+; Minimum time to force a plugin to wait before running again after a crash
+;min_penalty_sec = 30
+
+; Maximum time to force a plugin to wait after repeated crashes (8 hours default)
+;max_penalty_sec = 28800
+
+; If plugin runs successfully without crashing for this long, reset its
+; repeated error count
+;heal_threshold_sec = 300
+
+; Database processing rate limit. This will also be the rate at which design
+; documents are fetched. The rate is shared across all running plugins.
+;db_rate_limit = 50
+
+; Limits the rate at which plugins may open db shard files on a node. The
+; rate is shared across all running plugins.
+;shard_rate_limit = 50
+
+; Limit the rate at which plugins open documents. The rate is shared across all running plugins.
+;doc_rate_limit = 1000
+
+[couch_scanner_plugins]
+;couch_scanner_plugin_ddoc_features = false
```

Comment:
Rather than use boolean values here, could you instead use `enabled` and `disabled`, or is there more precedence for the former?

Reply:
I went with the idea that the whole section is a "Which plugins are enabled" section. So then each entry answers the question "is the plugin enabled = true / false?".

The precedent mostly all the boolean enabling flags we have `enable_database_recovery = false`, `utils_enable = true`. I don't think we have entries with `key = enable` pattern. There are some that are `enable = true | false` but those are singular features and here have a case where the key indicates what to enable.